buildpipeline/DotNet-CoreFx-Trusted-Windows.json

```diff
@@ -424,6 +424,9 @@
     },
     "PB_CleanAgent": {
       "value": "true"
+    },
+    "PB_SymbolsBuildId": {
+      "value": "DotNet-CoreFx-Windows"
```

Comment:
Adding System.DefinitionId would put us around 240 characters.  I think that's acceptable and makes sense.  I'll make that change now rather than wait and deal with it down the road.